sonar-csharp-plugin/src/main/java/org/sonar/plugins/csharp/CSharpPlugin.java

```diff
@@ -55,6 +57,7 @@ public void define(Context context) {
     context.addExtensions(
       METADATA,
       CSharp.class,
+      ProjectTypeCollector.class,
```

Comment:
_Out of curiosity:_ Does order of these types have any impact on their dependencies? Can we sort them?

Reply:
should be a different PR
I'll do it after